src/swap/components/SwapToast.tsx

```diff
@@ -1,17 +1,16 @@
-import { useCallback, useEffect, useMemo } from 'react';
-import { closeSvg } from '../../internal/svg/closeSvg';
-import { background, cn, color, text } from '../../styles/theme';
+import { useCallback } from 'react';
+import { cn, color, text } from '../../styles/theme';
 
 import { useAccount } from 'wagmi';
+import { Toast } from '../../internal/components/Toast';
 import { successSvg } from '../../internal/svg/successSvg';
-import { getToastPosition } from '../../internal/utils/getToastPosition';
 import { getChainExplorer } from '../../network/getChainExplorer';
 import type { SwapToastReact } from '../types';
 import { useSwapContext } from './SwapProvider';
 
 export function SwapToast({
   className,
-  durationMs = 3000,
+  durationMs = 8000,
```

Comment:
intended to leave?

Reply:
i did, but per comments, will revert to 5000ms